test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,210 @@
+
+			await expect(logo).toBeVisible();
+			const logoBox = await (await logo.elementHandle())?.boundingBox();
+
+			expect(logoBox?.height).toBeGreaterThan(0);
+			expect(logoBox?.width).toBeGreaterThan(0);
+
+			const title = app.code.driver.getLocator(titleLocator);
+
+			// product name in release is 'Positron' and in dev is 'Positron Dev'
+			await expect(title).toHaveText([/(Positron)|(Positron Dev)/, 'an IDE for data science']);
+
+			await expect(footer).toHaveText('Show welcome page on startup');
+		});
+
+		it('Verify Welcome page content', async function () {
+			const app = this.app as Application;
+
+			const startTitle = app.workbench.positronWelcome.startSection.getByRole('heading');
+			const startButtons = app.workbench.positronWelcome.startSection.getByRole('button');
+
+			const helpSection = app.code.driver.getLocator('.positron-welcome-page-help');
+			const helpTitle = helpSection.getByRole('heading');
+			const helpLinks = helpSection.getByRole('link');
+
+			const openSection = app.code.driver.getLocator('.categories-column.categories-column-right .index-list.start-container');
+			const openTitle = openSection.getByRole('heading');
+			const openButtons = openSection.getByRole('button');
+
+			const recentSection = app.code.driver.getLocator('.categories-column.categories-column-right .index-list.recently-opened');
+			const recentTitle = recentSection.getByRole('heading');
+
+			await expect(startTitle).toHaveText('Start');
+
+			await expect(startButtons).toHaveCount(4);
+			await expect(startButtons).toHaveText(['New Notebook', 'New File', 'New Console', 'New Project']);
+
+			await expect(helpTitle).toHaveText('Help');
+
+			await expect(helpLinks).toHaveCount(3);
+			await expect(helpLinks).toHaveText(['Positron Documentation', 'Positron Community', 'Report a bug']);
+
+			await expect(openTitle).toHaveText('Open');
+
+			await expect(openButtons).toHaveCount(3);
+			await expect(openButtons).toHaveText(['Open...', 'New Folder...', 'New Folder from Git...']);
+
+			await app.workbench.quickaccess.runCommand('File: Clear Recently Opened...');
+
+			await expect(recentTitle).toHaveText('Recent');
+
+			// 'open a folder' is a button so there is no character space because of its padding
+			await expect(recentSection.locator('.empty-recent')).toHaveText('You have no recent folders,open a folderto start.');
+		});
+
+		it('Click on new project from the Welcome page', async function () {
+			const app = this.app as Application;
+
+			await app.workbench.positronWelcome.clickNewProject();
+			await app.workbench.positronPopups.popupCurrentlyOpen();
+
+			await app.workbench.positronPopups.waitForModalDialogBox();
+
+			// confirm New Project dialog box is open
+			await app.workbench.positronPopups.waitForModalDialogTitle('Create New Project');
+
+			await app.workbench.positronPopups.clickCancelOnModalDialogBox();
+		});
+
+		describe('Python', () => {
+			before(async function () {
```

Comment:
All of these Python tests fail for me.. there is some issue in the startInterpreter used in the before(). Probably unrelated to this test.. I will need to investigate more.